data/gh2e/items.json

```diff
@@ -2,1369 +2,3504 @@
+            "small": true
+          }
+        ]
+      }
+    ],
     "noArtwork": true
   },
   {
     "id": 39,
     "name": "Ring of Skulls",
-    "cost": 0,
+    "cost": 40,
     "count": 2,
     "edition": "gh2e",
-    "slot": "unknown",
+    "slot": "small",
+    "consumed": true,
+    "unlockProsperity": 4,
+    "summon": {
+      "name": "skeleton",
+      "health": 3,
+      "attack": 2,
+      "movement": 2,
+      "range": 0
+    },
     "noArtwork": true
   },
   {
     "id": 40,
     "name": "Shadow Amor",
-    "cost": 0,
+    "cost": 25,
     "count": 2,
     "edition": "gh2e",
-    "slot": "unknown",
+    "slot": "body",
+    "consumed": true,
+    "unlockProsperity": 4,
+    "actions": [
+      {
+        "type": "custom",
+        "value": "%data.items.gh2e-40.1%",
+        "small": true
+      }
+    ],
+    "effects": [
+      {
+        "type": "element",
+        "value": "dark"
+      }
+    ],
     "noArtwork": true
   },
   {
     "id": 41,
     "name": "Kinetic Treads",
-    "cost": 0,
+    "cost": 30,
     "count": 2,
     "edition": "gh2e",
-    "slot": "unknown",
+    "slot": "legs",
+    "spent": true,
+    "unlockProsperity": 5,
+    "actions": [
+      {
+        "type": "custom",
+        "value": "%data.items.gh2e-41.1%",
+        "small": true
+      },
+      {
+        "type": "move",
+        "value": 2
+      }
+    ],
     "noArtwork": true
   },
   {
     "id": 42,
     "name": "Endurance Footwraps",
-    "cost": 0,
-    "count": 2,
-    "edition": "gh2e",
-    "slot": "unknown",
+    "cost": 40,
+    "count": 2,
+    "edition": "gh2e",
+    "slot": "legs",
+    "unlockProsperity": 5,
+    "actions": [
+      {
+        "type": "custom",
+        "value": "%data.items.gh2e-42.1%",
+        "small": true
+      },
+      {
+        "type": "heal",
+        "value": 1,
+        "subActions": [
+          {
+            "type": "specialTarget",
+            "value": "self",
+            "small": true
+          }
+        ]
+      }
+    ],
     "noArtwork": true
   },
   {
     "id": 43,
     "name": "Hawk Helm",
-    "cost": 0,
+    "cost": 20,
     "count": 2,
     "edition": "gh2e",
-    "slot": "unknown",
+    "slot": "head",
+    "spent": true,
+    "unlockProsperity": 5,
+    "actions": [
+      {
+        "type": "custom",
+        "value": "%data.items.gh2e-43.1%",
+        "small": true
+      }
+    ],
     "noArtwork": true
   },
   {
     "id": 44,
     "name": "Precision Bow",
-    "cost": 0,
-    "count": 2,
-    "edition": "gh2e",
-    "slot": "unknown",
+    "cost": 40,
+    "count": 2,
+    "edition": "gh2e",
+    "slot": "twohand",
+    "spent": true,
+    "unlockProsperity": 5,
+    "actions": [
+      {
+        "type": "custom",
+        "value": "%data.items.gh2e-44.1%",
+        "small": true
+      },
+      {
+        "type": "attack",
+        "value": 1,
+        "subActions": [
+          {
+            "type": "specialTarget",
+            "value": "targetAttackAbility",
```

Comment:
Please make this also a custom label. If you need target sign use `%game.action.target%`.